compiler/rustc_borrowck/src/diagnostics/mod.rs

```diff
@@ -175,144 +176,91 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
         place: PlaceRef<'tcx>,
         including_downcast: IncludingDowncast,
     ) -> Option<String> {
```

Comment:
GitHub won't let me comment on the documentation comment for this function, but it is written as if this function returns a `Result`, could you update it to reflect that the function returns `Option`?